adapters/taboola/taboola.go

```diff
@@ -176,10 +172,15 @@ func createTaboolaRequests(request *openrtb2.BidRequest) (taboolaRequests []*ope
 			errs = append(errs, err)
 			continue
 		}
-		if taboolaExt.TagId != "" {
-			imp.TagID = taboolaExt.TagId
-			modifiedRequest.Imp[i] = imp
+
+		tagId := taboolaExt.TagID
+		if len(taboolaExt.TagID) < 1 {
```

Comment:
this code has prioritisation of using taboolaExt.TagID even if both taboolaExt.TagId and taboolaExt.TagID are provided. Please update this in bidder document by raising docs PR.

Reply:
Yes - exactly 
`TagID` should be prioritized as the current logic. 
In the document we are mentioning only  it as the convention to pass tagid. However, the other param still in use as second option (deprecated ...)

Reply:
Have you created the docs PR to update the docs?

Reply:
What do I need to update the doc? 
As mentioned there publishers should pass the parameter as described there. 
in this PR we are aligning both Prebid.js and Prebid server adapter to use the same naming as in the official documentation

Reply:
In this [md file](https://github.com/prebid/prebid.github.io/blob/c9e93079ec99d3fd74d2f4d4043812b4a33de635/dev-docs/bidders/taboola.md), there is only one tagId defined.
In this PR, you are introducing new tagId. Although you have one of them deprecated but you should update this md file to reflect this change by creating a PR. Hope this helps.

Reply:
The "new" tagid is exactly the one in the documentation repo. It is a bug using the "old" one where used a different small letters tagid param for prebid server request till now (not following the documentation and Prebid.js  adapter). 
In this PR we are following and applying the documentation. However, we don't want harm live publishers that are already using the "deprecated" tagid and I don't think it's a good idea to add in the documentation anything about the "deprecated" tagid - this will mislead.

Reply:
ok. agreed that you don't wanna update the documentation to confuse existing or future publishers. Could we have more description added to `tagId` as mentioned [in this comment?](https://github.com/prebid/prebid-server/pull/2662#discussion_r1156770600)

Reply:
done